tests/unit/services/trigger-build-popup-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('service:trigger-build-popup', 'Unit | Service | trigger build popup', {
+  needs: ['service:auth']
+});
+
+test('it toggles the isShowingFlag', function (assert) {
+  let service = this.subject();
+  assert.equal(service.get('isShowingTriggerBuildModal'), false, 'initally the modal is hidden');
+  service.toggleTriggerBuildModal();
+  assert.equal(service.get('isShowingTriggerBuildModal'), true, 'after toggling the modal is visible');
+});
```

Comment:
IMO this is already covered by the acceptance test. But there’s no One True Way about which tests to rely on, so if you feel like this should be here, that’s fine too.